Automattic/jetpack
# Search: add filtering support Woo brands plugin

#### Changes proposed in this Pull Request:
The PR adds `product-brand` to taxonomy sync list, however this doesn't really have effects on syncing because all taxonomies are now synced. The change is to keep the list synced from WPCOM (`D86030-code`) and also make `Brands` available in Jetpack Search widget.

#### Other information:


#### Jetpack product discussion
p1660554618541599-jetpack-search-team-slack

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Open a site with Search plugin (and Search subscription), WooCommerce plugin and [WooCommerce Brands plugin](https://woocommerce.com/products/brands/)
- Open `/wp-admin/widgets.php` and add a Jetpack Search widget
- Ensure `Brands` is in the Taxonomy filter dropdown

![image](https://user-images.githubusercontent.com/1425433/185003062-fe207c72-2a88-484b-b13c-6f999f0c5100.png)
